common/app/views/support/HtmlCleaner.scala

```diff
@@ -292,6 +292,21 @@ case class LiveBlogDateFormatter(isLiveBlog: Boolean)(implicit val request: Requ
   }
 }
 
+case class BloggerBylineImage(article: Article)(implicit val request: RequestHeader) extends HtmlCleaner  {
+  def clean(body: Document): Document = {
+    if (article.isLiveBlog && LiveBlogContributorImagesSwitch.isSwitchedOn) {
+      body.select(".block").foreach { el =>
+        val contributorId = el.attributes().get("data-block-contributor")
+        article.tags.find(_.id == contributorId).map{ contributorTag =>
```

Comment:
Just check my solution actually works :smile: it came straight out of my head and never went through a compiler. Also I am assuming that `el.attributes().get("data-block-contributor")` returns `null` if there is no attribute (it is a Java library not a Scala one).